### DISCUSSION TITLE
CSP directives

### DISCUSSION THREAD

**Discussion by luffynando** [cawa-93/vite-electron-builder#115] [Q&A] [answered]
```html
<meta http-equiv="Content-Security-Policy" content="script-src 'self' 'unsafe-eval'">
``` 
vs
```html
<meta http-equiv="Content-Security-Policy" content="script-src 'self' blob:">
```

First option shows electron app warning security vs second works apparently good, what is advantages for use option one or option two?

**Comment by cawa-93** [comment, marked as answer]
First example was added after [this](https://github.com/cawa-93/vite-electron-builder/issues/109) and must be changed after [this](https://github.com/vitejs/vite/issues/2166). It's just that I haven't had time to make changes yet.
